pkgs/development/tools/esbuild/default.nix

```
@@ -0,0 +1,26 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+let
+  pname = "esbuild";
+  version = "0.11.12";
+
+in buildGoModule {
+  inherit pname version;
```

Could be simplified to:

```suggestion
buildGoModule rec {
  pname = "esbuild";
  version = "0.11.12";
```